superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -132,12 +134,23 @@ const CoverFooterRight = styled.div`
   text-overflow: ellipsis;
 `;
 
+const ThinSkeleton = styled(Skeleton)`
+  h3 {
+    margin: 5 0;
+  }
+
+  ul {
+    margin-bottom: 0;
+  }
+`;
+const paragraphConfig = { rows: 1, width: 150 };
```

Comment:
would some`%` or `vw` unit make more sense here, to be more responsive?

Reply:
I'll leave that for when we want card sizes to be responsive